src/config/networks/ganache.ts

```diff
@@ -0,0 +1,54 @@
+import { EnvironmentSettings, ETHEREUM_NETWORK, NetworkConfig } from 'src/config/networks/network.d'
+import { ETHGASSTATION_API_KEY } from '../../utils/constants'
+
+const baseConfig: EnvironmentSettings = {
+  txServiceUrl:
+    'http://localhost:8000/api/v1/transactions/0xeddf9e61fb9d8f5111840daef55e5fde0041f5702856532cdbb5a02998033d26/',
+  safeAppsUrl: 'http://localhost:3000',
```

Comment:
I'm not yet 100% sure I've set these values correctly. Perhaps, the Gnosis team can help guide here.